franco/Project/org.fundacionjala.automationproject/src/main/java/org/fundacionjala/automationproject/ui/FlightTicketPage.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.automationproject.ui;
+
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by Franco Aldunate on 11/20/2017.
+ */
+public class FlightTicketPage extends BasePage {
+  @FindBy(xpath = "/html/body/div/table/tbody/tr/td[2]"
```

Comment:
To concatenate string it is recommended to use String.format and not user "+"

Reply:
I reviewed it. I am only using this once and the attribute of xpath must be constant.